.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: node_js
 
+services:
+  - docker
+
+before_install:
+  - docker build /home/travis/build/scrum-gang/neutralinojs/ -t test
```

Comment:
I tried using relative paths (`.`) in my first commit but Travis didn't like it -- let me know if there's a better alternative to hardcoding this path. Maybe `pwd` may be an option but it feels less safe, and I don't think this path should ever change.

Reply:
I looked into your first failed build to see what was up. Turns out it's not working because before we run the `docker build`, we `cd neutralino.js` which is one folder deeper than the `Dockerfile`. I put a suggestion, but I'm not 100% sure that's the syntax

```suggestion
  - docker build ../. -t test
```

![image](https://user-images.githubusercontent.com/22057030/69481193-1b967480-0ddd-11ea-9935-2604f7a19914.png)

Reply:
Also we might want to change the tage to something more meaningful than `test`. Something as simple as `neutralinojs` could work

```suggestion
  - docker build /home/travis/build/scrum-gang/neutralinojs/ -t neutralinojs
```

Reply:
> we cd neutralino.js which is one folder deeper than the Dockerfile

In that case, using relative paths seems like a bad idea to me -- if we eventually choose to further change the path in which we are within travis, the docker part of the build will fail, but the docker part of the build should be independent of the other path-related operations we do. 

Ultimately, all docker should depend on is where the Dockerfile is -- so considering that we actively change directories I'd like to keep the fixed path. What do you think? I don't feel too strongly about it, but it feels less error-prone to use the fixed path in this case.

Reply:
I do agree with your point, but I did a bit more snooping around in your previous build and realized that you initially had your `docker build` in your `script`. That being said, doing `docker build . -t neutralinojs`  should work because it is done in the `before_install`  i.e. before and of the `cd`s are done. At the end of the day, I feel like it's the devs choice what to do. If you feel that there might be some directory change in the `before_install` as well, then keep the absolute path, otherwise I think relative pathing is just as effective here

Reply:
Sure, since I have it in the `before_install` now I'll keep it to relative paths ! Good call :+1: